src/Migrate/docs/Get-AzmigrateJob.md

```diff
@@ -0,0 +1,376 @@
+---
```

Comment:
Wrong casing in file name `Get-Az"m"igrateJob`

Reply:
Fixed